ui/src/app/pages/login/sign-in.tsx

```diff
@@ -289,15 +314,41 @@ export class SignInReactImpl extends React.Component<SignInProps, SignInState> {
     }
   }
 
+  handleCloseCookies() {
+    if (cookiesEnabled()) {
+      this.setState({cookieBannerClosed: true});
+      localStorage.setItem(cookieKey, 'cookie-banner-dismissed');
+    }
+  }
+
+  cookieBannerVisible() {
+    if (cookiesEnabled()) {
+      return !localStorage.getItem(cookieKey) && !this.state.cookieBannerClosed;
+    } else {
+      return true;
+    }
+  }
+
   render() {
     const backgroundImages = StepToImageConfig.get(this.state.currentStep);
     return <FlexColumn style={styles.signInContainer} data-test-id='sign-in-container'>
       <FlexColumn data-test-id='sign-in-page'
                   style={backgroundStyleTemplate(this.props.windowSize, backgroundImages)}>
         <div><img style={{height: '1.75rem', marginLeft: '1rem', marginTop: '1rem'}}
-                  src={HEADER_IMAGE}/></div>
+                  src={SIGNED_OUT_HEADER_IMAGE}/></div>
         {this.renderSignInStep(this.state.currentStep)}
       </FlexColumn>
+      {this.cookieBannerVisible() && <div style={styles.cookiePolicyMessage}>
+        <FlexRow style={{alignItems: 'center'}}>
+          <img src='assets/images/cookies.png'/>
+          <div style={{paddingLeft: '1rem', color: colors.primary}}>
+            We use cookies to help provide you with the best experience we can. By continuing to use our site, you consent
+            to our <a href='cookie-policy' target='_blank'
```

Comment:
[opt nit] should the href path have a preceding slash to make it an absolute path?